### PR TITLE
f-strings, line shortening, whitespace cleanup

### PR DESCRIPTION
**Pull request by ainen**
I've converted all ".format" to f-strings, shortened some lines, and cleaned up some whitespace

### CODE REVIEW

File: bot/models/BlockedLinks.py [hunk 1]
```diff
@@ -10,7 +10,10 @@ class BlockedLink(Base):
...
-    blocked_id = Column(Integer, ForeignKey('participants.id'), index=True, nullable=True)
+    blocked_id = Column(Integer, ForeignKey(
+        'participants.id'), index=True, nullable=True)
 
-    blocker = relationship("Participant", uselist=False, backref=backref('blocker_link'), foreign_keys=participant_id)
-    blocked = relationship("Participant", uselist=False, backref=backref('blocked_link'), foreign_keys=blocked_id)
+    blocker = relationship("Participant", uselist=False, backref=backref(
+        'blocker_link'), foreign_keys=participant_id)
+    blocked = relationship("Participant", uselist=False, backref=backref(
+        'blocked_link'), foreign_keys=blocked_id)
```
Comment (Voxter256) [inline bot/models/BlockedLinks.py]: I like the shortened line lengths! It feels weird to me to split the line not on one of the Column/relationship parameters, but inside the Foreign Key / backref functions inside those parameters.

They are long enough that It makes sense to just have a new line for each parameter.

ex: 
```
Column(
    Integer,
    ForeignKey('participants.id'),
    index=True,
    nullable=True
)
```

File: bot/models/Link.py [hunk 2]
```diff
@@ -3,14 +3,18 @@
...
 
+
 class Link(Base):
     __tablename__ = 'links'
 
     id = Column(Integer(), primary_key=True)
     santa_id = Column(Integer, ForeignKey('participants.id'), index=True)
-    receiver_id = Column(Integer, ForeignKey('participants.id'), index=True, default=None)
+    receiver_id = Column(Integer, ForeignKey(
+        'participants.id'), index=True, default=None)
```
Comment (Voxter256) [inline bot/models/Link.py]: Same as above; would prefer split before or after ForeignKey, not inside

File: bot/models/Link.py [hunk 3]
```diff
@@ -3,14 +3,18 @@
...
     group_id = Column(Integer, ForeignKey('groups.id'), index=True)
 
-    santa = relationship("Participant", uselist=False, backref=backref('link_santa'), foreign_keys=santa_id)
-    receiver = relationship("Participant", uselist=False, backref=backref('link_receiver'), foreign_keys=receiver_id)
+    santa = relationship("Participant", uselist=False,
+                         backref=backref('link_santa'), foreign_keys=santa_id)
+    receiver = relationship("Participant", uselist=False, backref=backref(
+        'link_receiver'), foreign_keys=receiver_id)
```
Comment (Voxter256) [inline bot/models/Link.py]: Same as above; would prefer split before or after backref, not inside

**Review by Voxter256 — CHANGES_REQUESTED** [review]
Thank you so much for all of the fixes! They look great! A couple of minor requests but I'll approve the pull after that!

**Comment by ainen** [comment]
I pushed the formatting changes to my fork

**Review by Voxter256 — APPROVED** [review]

**Pull request merged**